Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -227,15 +228,28 @@ public TestsRepository(TrappistDbContext dbContext, IGlobalUtil util, IStringCon
         {
             //Find the test by Id from Test Model
             var test = await _dbContext.Test.FindAsync(testId);
+           
             //Maps that test with TestAC
             var testAcObject = Mapper.Map<Test, TestAC>(test);
+            var testObject = await _dbContext.TestAttendees.FindAsync(testId);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'testObject' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)